.github/workflows/fossa-workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: fossa-workflow
+
+on:
```

Comment:
I wasn't aware of reserved key names in Yaml, but yamllint thinks this is a boolean. Since this is part of the Github Actions syntax the way I've found of ignoring this warning is with the following line:
`# yamllint disable-line rule:truthy`

This can either be on line 2, or come after `on:` (Ex: `on: # yamllint ...`
Here's a link to yamllint's documentation If you'd like to know more: https://yamllint.readthedocs.io/en/stable/disable_with_comments.html

Reply:
Thank you!.. I wasn't aware. let me take a look